core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -50,22 +48,12 @@ import org.apache.spark.util.{AccumulatorV2, Clock, SystemClock, Utils}
  *                        task set will be aborted
  */
 private[spark] class TaskSetManager(
-    sched: TaskSchedulerImpl,
+    val sched: TaskSchedulerImpl,
     val taskSet: TaskSet,
     val maxTaskFailures: Int,
-    clock: Clock = new SystemClock())
-  extends Schedulable with Logging {
+    val clock: Clock = new SystemClock()) extends Schedulable with Logging {
```

Comment:
not necessary, they were getting auto-promoted to fields anyway from they were used, I prefer to make that explicit.  I should have made them `private val`s, but I can drop it as well.